src/analysis/ContextBar.test.ts

```diff
@@ -554,8 +531,44 @@ describe('ContextBar - buttons', () => {
     expect(getByLabelText(new RegExp(/Cromwell Environment/i)));
   });
 
+  it('will not render a Cromwell button if workspace is not supported for cromwell', () => {
+    // Arrange
+    const cromwellOnAzureContextBarProps = {
+      ...contextBarPropsForAzure,
+      apps: [cromwellOnAzureRunning],
+      appDataDisks: [],
+    };
+
+    asMockedFn(doesWorkspaceSupportCromwellAppForUser).mockReturnValue(false);
+
+    // Act
+    const { getByLabelText, queryByLabelText } = render(h(ContextBar, cromwellOnAzureContextBarProps));
```

Comment:
I know I've said the opposite in the past, but i have since learned using queries from `screen` is the convention.
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen